cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/test_util_test.go

```diff
@@ -234,6 +238,59 @@ func installedRef(id, namespace string) *corev1.InstalledPackageReference {
 	}
 }
 
+func newCtrlClient(repos []sourcev1.HelmRepository, charts []sourcev1.HelmChart, releases []helmv2.HelmRelease) withWatchWrapper {
+	// register the flux GitOps Toolkit schema definitions
+	scheme := runtime.NewScheme()
+	_ = sourcev1.AddToScheme(scheme)
```

Comment:
Not sure why you're assigning to `_` here? Isn't that necessary only if you want some returned values but not others?

Reply:
just followed flux's example code:
https://fluxcd.io/docs/gitops-toolkit/packages/#crud-example